pageserver/src/metrics.rs

```diff
@@ -541,14 +541,25 @@ pub static SMGR_QUERY_TIME: Lazy<HistogramVec> = Lazy::new(|| {
     .expect("failed to define a metric")
 });
 
+// keep in sync with control plane Go code so that we can validate
+// compute's basebackup_ms metric with our perspective in the context of SLI/SLO.
+static COMPUTE_STARTUP_BUCKETS: Lazy<[f64; 28]> = Lazy::new(|| {
```

Comment:
Does look weird... To merge these, they'll have to be exact. Looking around `prometheus`

Reply:
Ok... Those really are floats, nevermind then.